app/src/androidTest/java/com/sdp/movemeet/Chat/ChatActivityTest.java

```diff
@@ -54,17 +64,31 @@ public void chatActivityTest() {
 
         onView(withId(R.id.edit_text_password)).perform(replaceText("234567"), closeSoftKeyboard());
 
-        onView(withId(R.id.button_login)).perform(click());
+        onView(withId(R.id.button_login)).perform(forceDoubleClick());
```

Comment:
why the button needs a double click ?

Reply:
We don't really understand, but sometimes the tests fail saying that the click to the button coordinate can't be done. And after some research, this is the solution that seems to be widespread to solve this problem. Maybe someone knows the reason ?